conf/configuration.yaml

```diff
@@ -72,17 +72,20 @@ arlas_auth:
   public_uris: [${ARLAS_AUTH_PUBLIC_URIS:-swagger,swagger.json}]
   certificate_file: ${ARLAS_AUTH_CERT_FILE:-/opt/app/arlas.pem}
 
-key_header: ${ARLAS_PERSISTENCE_KEY_HEADER:-X-Forwarded-User}
+
 persistence_engine: ${ARLAS_PERSISTENCE_ENGINE:-hibernate}
-firestore_collection: ${ARLAS_PERSISTENCE_FIRESTORE_COLLECTION:-}
+firestore_collection: ${ARLAS_PERSISTENCE_FIRESTORE_COLLECTION:-test}
+arlas_groups_header:  ${ARLAS_GROUPS_HEADER:-arlas_groups}
```

Comment:
We might set the HEADER env and forget to change the ARLAS_PERSISTENCE_CORS_ALLOWED_HEADERS...

Reply:
A check is apply on start and an exception is throwed if arlas header are missing in allowed headers in class ArlasPersistenceServerConfiguration.java